src/__tests__/components/DataDictionary.spec.tsx

```diff
@@ -26,15 +26,20 @@
 
 import React from 'react';
 import renderer from 'react-test-renderer';
-import {mockModels, mockCurrentModel, mockCurrentExplore} from "../MockData/MockData";
-
+import {mockModels, mockComments, mockCurrentModel, mockCurrentExplore} from "../MockData/MockData";
```

Comment:
you don't need to change this here, but you should get Prettier or some other linter running to catch and fix little things like this (no space after {)